js/webpack_get_defaults.js

```diff
@@ -68,7 +68,9 @@ function getOutputFileName () {
   let name = 'index.js'
   if (config.build.type === enums.buildTypes.APPLICATION) {
     name = 'bundle.js'
-    if (PRODUCTION_MODE) {
+    let enableFileNameHashing = config.build.enable_filename_hashing
+    enableFileNameHashing = enableFileNameHashing == null || enableFileNameHashing
```

Comment:
It'd be nicer to use a default value such as https://github.com/Polyconseil/systematic/blob/f4db19d16cb977da31bad5d8d4b17e5536656d85/js/config.js#L9